index.html

```diff
@@ -150,4 +151,4 @@ <h3>In this Tuturial you will learn about DOM Manipulation in JavaScript.</h3>
 
     </div>
```

Comment:
please use the div inside section to maintain the hierarchy, It make reading code much cleaner